subsys/bluetooth/host/hci_core.h

```diff
@@ -64,6 +65,36 @@ enum {
 #define BT_DEV_PERSISTENT_FLAGS (BIT(BT_DEV_ENABLE) | \
 				 BIT(BT_DEV_PRESET_ID))
 
+enum {
+	BT_ADV_CREATED,
+
+	BT_ADV_RPA_VALID,
+
+	BT_ADV_LIMITED,
+
+	BT_ADV_NUM_FLAGS,
+};
+
+struct bt_le_ext_adv {
+	/* ID Address used for advertising */
```

Comment:
I think it is a bit misleading to call it ID Address, which could be mistaken by the Device ID address. To me, this it is an ID. The core using the term advertising set. 
```suggestion
	/* ID used for an advertising set*/
```

Reply:
Would you change this or did you drop it?